tests/models/animals/test_animal_model.py

```diff
@@ -0,0 +1,80 @@
+"""Test module for model.py (and associated functionality).
+
+This module tests the functionality of model.py, as well as other bits of code that
+define models based on the class defined in model.py
+"""
```

Comment:
This is from the old base_model.py - best to briefly update it for the animal model.